EmployeeAdmin/src/com/controller/EmployeeStorage.java

```diff
@@ -0,0 +1,86 @@
+package com.controller;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.employee.DevEmployee;
+import com.employee.Employee;
+import com.employee.QAEmployee;
+
+/**
+ * This class implements the methods declared in IEmployeeStorage. 
+ * @author madalina.turcu
+ *
+ */
+
+public class EmployeeStorage implements IEmployeeStorage{
+
+	private static List<Employee> employeeList = new ArrayList<Employee>(){
+	    {
+	        add(new QAEmployee(1, "Mada", "mada@gmail.com", 23, "QA", true));
+	        add(new QAEmployee(2, "Ana", "ana@gmail.com", 21, "QA", false));
+	        add(new DevEmployee(3, "Eugen", "e@gmail.com", 21, "DEV", false));
+	        add(new DevEmployee(4, "Mircea", "m@gmail.com", 33, "DEV", true)); 
+	    }
+	 };
+	 
+	 /**
+	  * This method returns all the employees from the list.
+	  */
+	 public List<Employee> getAll() {
+		 return employeeList;
+	 }
+	 
+	 /**
+	  * This method returns a specific employee, found by id.
+	  */
+	 public Employee getById(int id) {
+		 
```

Comment:
there is no need to start with empty line here - really good thing that you are keeping the code clean and easy to read, but you don't have to start with an empty line - same for the other methods.